tests/functional/idempotency/test_idempotency.py

```diff
@@ -61,7 +61,7 @@
 from tests.functional.utils import json_serialize, load_event
 
 TABLE_NAME = "TEST_TABLE"
-TESTS_MODULE_PREFIX = "test-func.functional.idempotency.test_idempotency"
+TESTS_MODULE_PREFIX = "test-func.tests.functional.idempotency.test_idempotency"
```

Comment:
I added the `__init__.py` file in the `tests` folder, so Python is now marking this directory as a package. I need this to enable nox import utility functions and cross-reference files, otherwise it will fail.